packages/calcite-components/src/utils/globalScript.ts

```diff
@@ -1,18 +1,18 @@
+import { isServer } from "lit";
 import { initModeChangeEvent } from "./mode";
 import { stampVersion } from "./config";
-import { isBrowser } from "./browser";
 
 /**
  * This file is imported in Stencil's `globalScript` config option.
  *
  * @see [Stencil's globalScript](https://stenciljs.com/docs/config#globalscript).
  */
 export default function (): void {
-  if (isBrowser()) {
-    if (document.readyState === "interactive") {
+  if (!isServer) {
+    if (document?.readyState === "interactive") {
```

Comment:
Can `document` be undefined in browser contexts?

Reply:
No, I was just optional chaining all instances of `document` in case the `isServer` constant didn't work like Stencil's. But I can undo that